Group6PublicLibrary/src/group6ranger/RangeSensor.java

```diff
@@ -0,0 +1,145 @@
+package group6ranger;
```

Comment:
Again make sure you rename your packages to follow good Java package naming convention in your public repo as well.